CHANGES.md

```diff
@@ -2,6 +2,8 @@
 
 * Remove legacy `ReactDOMRe` and `ReasonReact` modules (@anmonteiro in
   [#782](https://github.com/reasonml/reason-react/pull/782))
+* Fix `React.useCallback*` for callbacks with multiple arguments (@anmonteiro
```

Comment:
Should this be now "for uncurried callbacks"?

Reply:
I think it's both, so best to leave it unspecified.